lib/cli.js

```diff
@@ -71,41 +74,64 @@ module.exports.run = function(argv, callback) {
       configFile: configFile,
       fix: fix,
       files: files
-    }, function(err, result) {
+    }, function (err, result) {
       if (err) {
         return callback(err);
       }
-
+      if (!result.errorCount && !result.warningCount) {
+        return callback(null, Chalk.green.bold('\nClean as a whistle! No issues found!\n'), 0);
+      }
       formatResults(result, callback);
     });
   });
 };
 
 
-function formatResults(results, callback) {
-  var totalWarnings = results.warningCount;
-  var totalErrors = results.errorCount;
+function formatResults (result, callback) {
+  var totalWarnings = result.warningCount;
+  var totalErrors = result.errorCount;
   var exitCode = +!!totalErrors;
-  var output = '';
+  var output = '\n';
+  var position = Chalk.gray.bold;
+  var error = Chalk.red;
+  var warning = Chalk.yellow;
+  var colorCode = {
+    2: error,
+    1: warning
+  };
+
+  result.results.filter(function (item) {
+    return item.errorCount || item.warningCount;
+  }).forEach(function (file) {
+    output += 'Problems in: ' + Chalk.dim(file.filePath) + '\n';
+
+    // Sort by severity, 2's first because they are errors
```

Comment:
Seems like this would make it harder to go through and fix issues since you could potentially be jumping all over the file. I think ordering by line number would be best.